base_layer/core/src/transactions/transaction_components/transaction_output.rs

```diff
@@ -575,4 +608,22 @@ mod test {
             1024, 1024, 1024, 1024, 1024, 1024, 1024, 1024, 1024, 1024, 64, 2, 1
         ]);
     }
+
+    #[test]
+    fn it_builds_correctly_from_unblinded_output() {
```

Comment:
Would also be good to have the failing case where the output commitment amount is less than the stated minimum